bin/check_conda_package_versions.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
```

Comment:
The script itself needs to be checked-in with executable permissions.

Reply:
Done